### PR TITLE
Radio migration guide

### PR DESCRIPTION
**Pull request by ericos**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)

**Description:**

Adds info about the `Radio` and `RadioGroup` widgets to the migration guide. 

Part of #1242

### CODE REVIEW

File: docs/V7-Migration-Guide.md [hunk 1]
```diff
@@ -355,21 +355,9 @@ Latest example can be found on [widgets.dojo.io/#widget/progress/overview](https
 
 ### radio
 
-#### Property changes
```
Comment (tomdye) [inline docs/V7-Migration-Guide.md:358]: why did you remove all these headers? I think label after needs to go in `changed properties` ?

File: docs/V7-Migration-Guide.md [hunk 2]
```diff
@@ -356,20 +356,21 @@ Latest example can be found on [widgets.dojo.io/#widget/progress/overview](https
...
-##### Changed properties
-- bar: string
-	- this prop replaced x
-	- this prop does foo bar baz
-	- more info
 ##### Removed properties
-- baz: string
-	- replaced by foo
-	- any additional info
+- label?: () => RenderResult
```
Comment (tomdye) [inline docs/V7-Migration-Guide.md:360]: this is just `RenderResult`

Comment (ericos) [reply docs/V7-Migration-Guide.md:360]: Yep, will get this updated with the latest changes from this week.
